### PR TITLE
add docs  delete post

### PR DESCRIPTION
**Pull request by ghost**
## Description
docs delete post 
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
 YES

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

### CODE REVIEW

File: database/migrations/2018_06_14_025528_change_rate_and_content_column.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('posts', function (Blueprint $table) {
+            $table->string('rate')->nullable(true)->change();
```
Comment (at-dungle) [inline database/migrations/2018_06_14_025528_change_rate_and_content_column.php]: nullable(true) là sao?

**Comment by at-thaitran** [comment]
LGTM

**Comment by at-thaitran** [comment]
LGTM

**Comment by at-thinhnguyen** [comment]
xoá thành công thì ko cần response content trả về, status code =200, xoá fail thì trả về reponse content gì, status code nữa

**Pull request merged**